# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: 967e68ed5f56
# Full Hash: 967e68ed5f56f40a159da9664a1f789d32cff134
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 3
# Description:
#   Bug 1742437 - move DOM script context into own file and rename it; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138132
# ==============================================================================

diff -r 22649272ffae -r 967e68ed5f56 dom/script/moz.build
--- a/dom/script/moz.build	Fri Feb 18 12:45:51 2022 +0000
+++ b/dom/script/moz.build	Fri Feb 18 12:45:52 2022 +0000
@@ -26,6 +26,7 @@
     "ScriptDecoding.h",
     "ScriptElement.h",
     "ScriptKind.h",
+    "ScriptLoadContext.h",
     "ScriptLoader.h",
     "ScriptLoadRequest.h",
     "ScriptSettings.h",
@@ -40,6 +41,7 @@
     "ModuleMapKey.cpp",
     "nsIScriptElement.cpp",
     "ScriptElement.cpp",
+    "ScriptLoadContext.cpp",
     "ScriptLoader.cpp",
     "ScriptLoadHandler.cpp",
     "ScriptLoadRequest.cpp",
